upnotify-bot/src/main/java/utils/WebUtils.java

```diff
@@ -127,6 +130,24 @@ public String getStringFromUrlAndSelectorPathJsoup(String url, String selectorPa
 		Element el = getElementFromUrlAndSelectorPathJsoup(url, selectorPath);
 		return el.text();
 	}
+
+	public String getStringFromUrlAndSelectorPathUsingJsoupAndSelenium(String url, String selectorPath) {
+		//URL driverURL = getClass().getClassLoader().getResource("chromedriver.exe");
+		//System.out.println("URL:" + driverURL);
+		System.setProperty("webdriver.chrome.driver", "src/main/resources/chromedriver.exe");
+		WebDriver driver = new ChromeDriver();
+		driver.get(url);
+		Document doc = Jsoup.parse(driver.getPageSource());
+		Elements elements = doc.select(selectorPath);
+		Element el = elements.get(0);
+		return el.toString();
+	}
+
+	public void getScreenshotUsingSelenium(String url, String selectorPath) {
+		System.setProperty("webdriver.chrome.driver", "src/main/resources/chromedriver.exe");
```

Comment:
Try to use something like "ClassLoader.getSystemResourceAsStream" instead of typing the whole path "src/main/resources/chromedriver.exe" yourself.